mne/__init__.py

```diff
@@ -98,6 +98,7 @@
 from . import viz
 from . import decoding
 from . import realtime
+from . import testing
```

Comment:
+1 for common.py

and +1 not to rush this merge before some extended test and validation.

didn't you already add the mne_map_data?